rerun_cpp/src/rerun/recording_stream.hpp

```diff
@@ -129,13 +129,27 @@ namespace rerun {
         /// that viewer instead of starting a new one.
         ///
         /// ## Parameters
+        /// options:
+        /// See `rerun::SpawnOptions` for more information.
         ///
         /// flush_timeout_sec:
         /// The minimum time the SDK will wait during a flush before potentially
         /// dropping data if progress is not being made. Passing a negative value indicates no
         /// timeout, and can cause a call to `flush` to block indefinitely.
         Error spawn(const SpawnOptions& options = {}, float flush_timeout_sec = 2.0) const;
 
+        /// @see RecordingStream::spawn
+        template <typename TRep, typename TPeriod>
+        Error spawn(
+            const SpawnOptions& options = {},
+            std::chrono::duration<TRep, TPeriod> flush_timeout = std::chrono::seconds(2)
+        ) const {
+            return spawn(
+                options,
+                std::chrono::duration_cast<std::chrono::duration<float>>(flush_timeout).count()
```

Comment:
Maybe a comment here reminding non-chrono experts that this defaults to a std::ratio<1,1> => seconds

Reply:
ah yeah that is weird isn't it. Also commenting that in elsewhere probably!